server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -384,11 +384,11 @@ MetadataTime computeRootTabletTime(ServerContext context, Collection<String> goo
         Path path = new Path(good);
 
         FileSystem ns = context.getVolumeManager().getFileSystemByPath(path);
+        var tableConf = context.getTableConfiguration(RootTable.ID);
         long maxTime = -1;
         try (FileSKVIterator reader = FileOperations.getInstance().newReaderBuilder()
-            .forFile(path.toString(), ns, ns.getConf(), context.getCryptoService())
-            .withTableConfiguration(context.getTableConfiguration(RootTable.ID)).seekToBeginning()
-            .build()) {
+            .forFile(path.toString(), ns, ns.getConf(), tableConf.getCryptoService())
```

Comment:
Good question. I am not sure. I think it would be simpler to assume no encryption. But if the user wants the files to be encrypted during the upgrade, I think they would need the service here.

Reply:
And, I think I forgot when I commented that encryption existed in the prior version. So, the file could be encrypted already using the prior implementation?

Reply:
Good point. We could add a method to the Upgrader to check for encryption. But that could be a follow on and maybe just a nice to have.